File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.c
#include<stdio.h>

int main(void){
    int n,num;
    scanf("%d",&n);
    int arr[10001] = {0,};
    for(int i = 0; i < n; i++){
        scanf("%d",&num);
        arr[num] = arr[num] + 1;
    }
    for(int i = 1; i < 10001; i++){
        for(int j = 1; j <= arr[i]; j++){
            printf("%d\n",i);
        }
    }
    return 0;
}

File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))
    val sb = StringBuilder()

    val n = br.readLine().toInt()
    val arr = IntArray(n)

    for (i in 0 until n) {
        arr[i] = br.readLine().toInt()
    }
    arr.sort()

    for (num in arr) {
        sb.append(num).append("\n")
    }
    bw.write(sb.toString())
    bw.flush()
    bw.close()
}